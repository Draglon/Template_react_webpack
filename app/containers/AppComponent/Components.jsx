// Pages
import Home from '../Home/index.jsx';
import Contacts from '../Contacts/index.jsx';
import NotFound from '../NotFound/index.jsx';

export {
  Home,
  Contacts,
  NotFound,
};
