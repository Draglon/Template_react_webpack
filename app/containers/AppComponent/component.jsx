import React from 'react';
import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';

import Header from '../Header';

import Home from '../Home';
import NotFound from '../NotFound';

const AppComponent = () => (
  <Router>
    <main className="main page">
      <Header />
      <Switch>
        <Route exact path="/" component={Home} />
        <Route component={NotFound} />
      </Switch>
    </main>
  </Router>
);

export default AppComponent;
