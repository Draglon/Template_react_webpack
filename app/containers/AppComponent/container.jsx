import { connect } from 'react-redux';
import { accessAdd } from '../../store/theMovieDB/auth/actions';

import AppComponent from './component';

const mapDispatchToProps = {
  accessAdd,
};

export default connect(
  null,
  mapDispatchToProps,
)(AppComponent);
