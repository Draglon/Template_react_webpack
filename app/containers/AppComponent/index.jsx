import AppComponent from './component';

export default AppComponent;
