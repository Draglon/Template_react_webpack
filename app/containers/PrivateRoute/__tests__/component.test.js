import React from 'react';
import toJson from 'enzyme-to-json';

import AppComponent from '../component';

describe('<AppComponent /> component', () => {
  let component;

  beforeEach(() => {
    component = shallow(<AppComponent />);
  });

  it('Render property', () => {
    expect(toJson(component)).toMatchSnapshot();
  });
});
