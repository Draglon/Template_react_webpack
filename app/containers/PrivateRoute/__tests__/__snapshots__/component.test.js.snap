// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppComponent /> component Render property 1`] = `
<Route
  render={[Function]}
/>
`;
